### PR TITLE
DT-1055: 🔨 Add in 100.64/10 address range for internal proxies and log

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonestate/OpenApiConfiguration.kt [hunk 1]
```diff
@@ -21,15 +20,5 @@ class OpenApiConfiguration(buildProperties: BuildProperties) {
           )
               .contact(Contact().name("HMPPS Digital Studio").email("feedback@digital.justice.gov.uk"))
       )
-
-  @Bean
-  fun logFilter(): CommonsRequestLoggingFilter {
```
Comment (petergphillips) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonestate/OpenApiConfiguration.kt:26]: removed as don't need that information any more

File: src/main/resources/logback-spring.xml [hunk 2]
```diff
@@ -59,7 +59,7 @@
     <appender-ref ref="logAppender"/>
   </logger>
 
-  <logger name="org.springframework.web.filter.CommonsRequestLoggingFilter" level="DEBUG">
+  <logger name="org.apache.catalina.valves.RemoteIpValve" level="DEBUG">
```
Comment (petergphillips) [inline src/main/resources/logback-spring.xml:62]: log the remote ip valve instead to see what's going on

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -38,6 +38,7 @@ server:
     remoteip:
       remote-ip-header: x-forwarded-for
       protocol-header: x-forwarded-proto
+      internal-proxies: 10\.\d{1,3}\.\d{1,3}\.\d{1,3}|192\.168\.\d{1,3}\.\d{1,3}|169\.254\.\d{1,3}\.\d{1,3}|127\.\d{1,3}\.\d{1,3}\.\d{1,3}|172\.1[6-9]{1}\.\d{1,3}\.\d{1,3}|172\.2[0-9]{1}\.\d{1,3}\.\d{1,3}|172\.3[0-1]{1}\.\d{1,3}\.\d{1,3}|0:0:0:0:0:0:0:1|::1|100\.6[4-9]\.\d{1,3}\.\d{1,3}|100\.[7-9][0-9]{1}\.\d{1,3}\.\d{1,3}|100\.1[0-1][0-9]{1}\.\d{1,3}\.\d{1,3}|100\.12[0-7]\.\d{1,3}\.\d{1,3}
```
Comment (petergphillips) [inline src/main/resources/application.yml:41]: This is expanded into
```
10\.\d{1,3}\.\d{1,3}\.\d{1,3}
192\.168\.\d{1,3}\.\d{1,3}
169\.254\.\d{1,3}\.\d{1,3}
127\.\d{1,3}\.\d{1,3}\.\d{1,3}
172\.1[6-9]{1}\.\d{1,3}\.\d{1,3}
172\.2[0-9]{1}\.\d{1,3}\.\d{1,3}
172\.3[0-1]{1}\.\d{1,3}\.\d{1,3}
0:0:0:0:0:0:0:1
::1
100\.6[4-9]\.\d{1,3}\.\d{1,3}
100\.[7-9][0-9]{1}\.\d{1,3}\.\d{1,3}
100\.1[0-1][0-9]{1}\.\d{1,3}\.\d{1,3}
100\.12[0-7]\.\d{1,3}\.\d{1,3}
```
so the top lot are the default in the RemoteIpValve and the bottom lot are 100.64/10, which translates to 100.64.0.0 -> 10.127.255.255

**Review by Mjwillis — APPROVED** [review]
Horrible but LGTM

**Pull request merged**